sdk/spring/spring-integration-azure-eventhubs/src/test/java/com/azure/spring/integration/eventhubs/inbound/EventHubsInboundChannelAdapterTests.java

```diff
@@ -116,15 +116,15 @@ void setInstrumentationId() {
 
     @Test
     void setMessageConverter() {
-        AbstractAzureMessageConverter<EventData, EventData> converter = mock(EventHubsMessageConverter.class);
+        AbstractJacksonAzureMessageConverter<EventData, EventData> converter = mock(EventHubsMessageConverter.class);
         this.adapter.setMessageConverter(converter);
         assertThat(this.adapter).extracting("recordListener").extracting("messageConverter").isEqualTo(converter);
         assertThat(this.adapter).extracting("batchListener").extracting("messageConverter").isNotEqualTo(converter);
     }
 
     @Test
     void setBatchMessageConverter() {
-        AbstractAzureMessageConverter<EventBatchContext, EventData> converter = mock(EventHubsBatchMessageConverter.class);
+        AbstractJacksonAzureMessageConverter<EventBatchContext, EventData> converter = mock(EventHubsBatchMessageConverter.class);
```

Comment:
same

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-java/pull/32721